frontend/build/embedding-sdk/utils/update-chunk-content-and-source-map.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from "fs";
+
+import { SourceMapConsumer, SourceMapGenerator } from "source-map";
+
+import { ENABLE_SOURCE_MAPS } from "../constants/source-maps-enabled.mjs";
+
+export async function updateChunkContentAndSourceMap(filePath, updateContent) {
```

Comment:
Oh wow, I guess `esbuild` does not support source maps by default and so we have to do source mapping by ourselves?

Reply:
It supports. But we have to do some post-build adjustments and there this helper is used